pkg/tcpip/stack/ndp.go

```diff
@@ -284,6 +300,12 @@ type NDPConfigurations struct {
 	// processed.
 	HandleRAs bool
 
+	// HandleRSs determines whether or not Router Advertisements will be
+	// processed.
+	HandleRSs bool
+
+	routerConfig NDPRouterConfiguration
```

Comment:
This field should be exported so users of this type (from outside this package) can set the router configurations.